sql/core/src/main/scala/org/apache/spark/sql/execution/SparkSqlParser.scala

```diff
@@ -751,15 +751,15 @@ class SparkSqlAstBuilder extends AstBuilder {
           (Nil, Option(name), props, recordHandler)
       }
 
-      val (inFormat, inSerdeClass, inSerdeProps, reader) =
+      val (inFormat, inSerdeClass, inSerdeProps, writer) =
```

Comment:
Maybe we can add some comments to show why we write code like this.

Reply:
ok